src/main/steps/common/page.njk

```diff
@@ -35,14 +37,17 @@
     },
     html: feedback + languageToggle
   }) }}
-  {% block backLink %}
```

Comment:
@AbdullahShahHMCTS We shouldn't remove this existing back link, it will impact respondent journey screens.
We need to make sure don't break anything which is present & working fine.